CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # App Center SDK for Unity Change Log
 
+## Version 2.6.1 (Under development)
+
+### App Center Crashes
+
+#### iOS
+
+* **[Fix]** Fix that Crashes could not be started because `StartCrashes` method was stripped for a high managed stripping level.
```

Comment:
```suggestion
* **[Fix]** Fix an issue where Crashes could not be started because `StartCrashes` method was stripped for a high managed stripping level.
```